features-json/web-app-manifest.json

```
@@ -398,7 +398,8 @@
   "notes_by_num":{
     "1":"The installation events, `appinstalled` and `beforeinstallprompt`, are not supported.",
     "2":"The installation events, `appinstalled` and `beforeinstallprompt`, are not supported on all platforms.",
-    "3":"A manifest could be used to list apps in the Microsoft Store, which would use Edge as a back-end."
+    "3":"A manifest could be used to list apps in the Microsoft Store, which would use Edge as a back-end.",
+    "4":"Safari on IOS does not support A2HS in WebViews like Chrome and Firefox"
```

tiny improvement:
```suggestion
    "4":"Safari on iOS does not support A2HS in WebViews like Chrome and Firefox"
```

0:-)

https://en.wikipedia.org/wiki/Cisco_IOS :P

---

Fair 😄